### PR TITLE
Staking updates

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -1206,33 +1206,50 @@
...
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
+            },
+            "sections": {
+                "about": "자세한 내용",
+                "balances": "Balances",
+                "stakes": "Stakes"
+            },
```
Comment (vetkamak) [inline src/ko.json]: @kkkhesha could you translate it, please? :)

File: src/ru.json [hunk 2]
```diff
@@ -1195,33 +1195,50 @@
...
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
+            },
+            "sections": {
+                "about": "Информация",
+                "balances": "Balances",
+                "stakes": "Stakes"
+            },
```
Comment (vetkamak) [inline src/ru.json]: @AnnStepanova could you translate these strings, pls? :)

File: src/ko.json [hunk 3]
```diff
@@ -1206,33 +1206,50 @@
...
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
+            },
+            "sections": {
+                "about": "자세한 내용",
+                "balances": "Balances",
+                "stakes": "Stakes"
```
Comment (kkkhesha) [inline src/ko.json:1241]:
```suggestion
                "stakes": "Stakes"
```
```suggestion
                "stakes": "스테이킹"
```

File: src/ko.json [hunk 4]
```diff
@@ -1206,33 +1206,50 @@
...
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
+            },
+            "sections": {
+                "about": "자세한 내용",
+                "balances": "Balances",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "balances": "잔액",
```

File: src/ko.json [hunk 5]
```diff
@@ -1206,33 +1206,50 @@
...
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
```
Comment (kkkhesha) [inline src/ko.json:1236]:
```suggestion
                "withdraw": "Withdraw"
```
```suggestion
                "withdraw": "출금"
```

File: src/ko.json [hunk 6]
```diff
@@ -1206,33 +1206,50 @@
...
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
```
Comment (kkkhesha) [inline src/ko.json:1235]:
```suggestion
                "info": "How does it work?",
```
```suggestion
                "info": "어떻게 작동하나요?",
```

File: src/ko.json [hunk 7]
```diff
@@ -1206,33 +1206,50 @@
...
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
```
Comment (kkkhesha) [inline src/ko.json:1234]:
```suggestion
                "deposit": "Deposit",
```
```suggestion
                "deposit": "예금",
```

File: src/ko.json [hunk 8]
```diff
@@ -1206,33 +1206,50 @@
...
+                "availablePool": "사용 가능한 풀",
+                "assurance": "보증금",
+                "minStake": "최소 스테이킹 금액",
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "withdrawal": "출금"
```

File: src/ko.json [hunk 9]
```diff
@@ -1206,33 +1206,50 @@
...
+                "rewardPaid": "적립된 보상",
+                "availablePool": "사용 가능한 풀",
+                "assurance": "보증금",
+                "minStake": "최소 스테이킹 금액",
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "available": "사용 가능",
```

File: src/ko.json [hunk 10]
```diff
@@ -1206,33 +1206,50 @@
...
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "수명 (age)",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "age": "수명",
```

File: src/ko.json [hunk 11]
```diff
@@ -1206,33 +1206,50 @@
...
-            "availablePool": "사용 가능한 풀",
-            "balance": "잔액",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "validatorFee": "밸리데이터 수수료",
```

File: src/ko.json [hunk 12]
```diff
@@ -1206,33 +1206,50 @@
...
-            "assurance": "보증금",
-            "availablePool": "사용 가능한 풀",
-            "balance": "잔액",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "currentAPY": "현재 APY",
```

File: src/ko.json [hunk 13]
```diff
@@ -1206,33 +1206,50 @@
...
-            "about": "자세한 내용",
-            "advancedFeatures": "고급 기능",
-            "age": "수명 (age)",
-            "assurance": "보증금",
-            "availablePool": "사용 가능한 풀",
-            "balance": "잔액",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "reward": "+{0} % 또는 {1} {2}"
```

File: src/ko.json [hunk 14]
```diff
@@ -1206,33 +1206,50 @@
...
         },
         "stakingScreen": {
-            "about": "자세한 내용",
-            "advancedFeatures": "고급 기능",
-            "age": "수명 (age)",
-            "assurance": "보증금",
-            "availablePool": "사용 가능한 풀",
-            "balance": "잔액",
+            "balance": {
+                "title": "Surf stakes in {0}",
```
Comment (kkkhesha) [inline src/ko.json:1210]:
```suggestion
                "title": "Surf stakes in {0}",
```
```suggestion
                "title": "{0}에 서프 스테이킹",
```

Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "title": "{0}에 서프 스테이킹",
```

File: src/ko.json [hunk 15]
```diff
@@ -1206,33 +1206,50 @@
...
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "efficiency": "효율성",
```

File: src/ko.json (hunk 7 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "deposit": "예금",
```

File: src/ko.json (hunk 6 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "info": "어떻게 작동하나요?",
```

File: src/ko.json (hunk 3 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "stakes": "스테이킹"
```

File: src/ko.json (hunk 5 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "withdraw": "출금"
```

File: src/ko.json [hunk 16]
```diff
@@ -1206,33 +1206,50 @@
...
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "적립된 보상",
+                "availablePool": "사용 가능한 풀",
+                "assurance": "보증금",
+                "minStake": "최소 스테이킹 금액",
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "staked": "스테이킹된 금액",
```

File: src/ko.json [hunk 17]
```diff
@@ -1206,33 +1206,50 @@
...
         "stakingScreen": {
-            "about": "자세한 내용",
-            "advancedFeatures": "고급 기능",
-            "age": "수명 (age)",
-            "assurance": "보증금",
-            "availablePool": "사용 가능한 풀",
-            "balance": "잔액",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "note": "전체 시간 동안 {0}",
```

File: src/ko.json [hunk 18]
```diff
@@ -1206,33 +1206,50 @@
...
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "수명 (age)",
+                "members": "회원",
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "적립된 보상",
+                "availablePool": "사용 가능한 풀",
+                "assurance": "보증금",
+                "minStake": "최소 스테이킹 금액",
+                "operationDeposit": "Operation deposit",
```
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
                "operationDeposit": "작업 예금",
```

File: src/ru.json [hunk 19]
```diff
@@ -1195,33 +1195,50 @@
...
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
+            },
+            "sections": {
+                "about": "Информация",
+                "balances": "Balances",
+                "stakes": "Stakes"
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "stakes": "Стейки"
```

File: src/ru.json [hunk 20]
```diff
@@ -1195,33 +1195,50 @@
...
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
+            },
+            "sections": {
+                "about": "Информация",
+                "balances": "Balances",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "balances": "Баланс",
```

File: src/ru.json [hunk 21]
```diff
@@ -1195,33 +1195,50 @@
...
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
+                "withdraw": "Withdraw"
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "withdraw": "Вывести"
```

File: src/ru.json [hunk 22]
```diff
@@ -1195,33 +1195,50 @@
...
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
+                "info": "How does it work?",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "info": "Как это работает?",
```

File: src/ru.json [hunk 23]
```diff
@@ -1195,33 +1195,50 @@
...
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
+            },
+            "buttons": {
+                "deposit": "Deposit",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "deposit": "Разместить",
```

File: src/ru.json [hunk 24]
```diff
@@ -1195,33 +1195,50 @@
...
+                "availablePool": "Доступный пул",
+                "assurance": "Гарантия",
+                "minStake": "Мин. ставка",
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
+                "withdrawal": "Withdrawal"
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "withdrawal": "Выведено"
```

File: src/ru.json [hunk 25]
```diff
@@ -1195,33 +1195,50 @@
...
+                "rewardPaid": "Вознаграждение выплачено",
+                "availablePool": "Доступный пул",
+                "assurance": "Гарантия",
+                "minStake": "Мин. ставка",
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
+                "available": "Available",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "available": "Доступно",
```

File: src/ru.json [hunk 26]
```diff
@@ -1195,33 +1195,50 @@
...
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "Вознаграждение выплачено",
+                "availablePool": "Доступный пул",
+                "assurance": "Гарантия",
+                "minStake": "Мин. ставка",
+                "operationDeposit": "Operation deposit",
+                "percentage": "{0} %"
+            },
+            "balancesDetails": {
+                "staked": "Staked",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "staked": "Размещено",
```

File: src/ru.json [hunk 27]
```diff
@@ -1195,33 +1195,50 @@
...
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "Возраст",
+                "members": "Участники",
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "Вознаграждение выплачено",
+                "availablePool": "Доступный пул",
+                "assurance": "Гарантия",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "assurance": "Страховка",
```

File: src/ru.json [hunk 28]
```diff
@@ -1195,33 +1195,50 @@
...
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "Возраст",
+                "members": "Участники",
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "Вознаграждение выплачено",
+                "availablePool": "Доступный пул",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "availablePool": "Вместимость",
```

File: src/ru.json [hunk 29]
```diff
@@ -1195,33 +1195,50 @@
...
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "Возраст",
+                "members": "Участники",
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "Вознаграждение выплачено",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "rewardPaid": "Выплачено участникам",
```

File: src/ru.json [hunk 30]
```diff
@@ -1195,33 +1195,50 @@
...
-            "availablePool": "Доступный пул",
-            "balance": "Баланс",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "validatorFee": "Комиссия валидатора",
```

File: src/ru.json [hunk 31]
```diff
@@ -1195,33 +1195,50 @@
...
-            "assurance": "Гарантия",
-            "availablePool": "Доступный пул",
-            "balance": "Баланс",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "currentAPY": "Доходность",
```

File: src/ru.json [hunk 32]
```diff
@@ -1195,33 +1195,50 @@
...
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "Возраст",
+                "members": "Участники",
+                "tvl": "TVL,  {0}",
+                "rewardPaid": "Вознаграждение выплачено",
+                "availablePool": "Доступный пул",
+                "assurance": "Гарантия",
+                "minStake": "Мин. ставка",
+                "operationDeposit": "Operation deposit",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "operationDeposit": "Комиссия контракта",
```

File: src/ru.json [hunk 33]
```diff
@@ -1195,33 +1195,50 @@
...
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
+                "age": "Возраст",
+                "members": "Участники",
+                "tvl": "TVL,  {0}",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "tvl": "Текущий баланс,  {0}",
```

File: src/ru.json [hunk 34]
```diff
@@ -1195,33 +1195,50 @@
...
-            "balance": "Баланс",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
+            },
+            "aboutDetails": {
+                "currentAPY": "Current APY",
+                "validatorFee": "Validator fee",
+                "efficiency": "Efficiency",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "efficiency": "Надежность",
```

File: src/ru.json [hunk 35]
```diff
@@ -1195,33 +1195,50 @@
...
-            "about": "Информация",
-            "advancedFeatures": "Расширенные функции",
-            "age": "Возраст",
-            "assurance": "Гарантия",
-            "availablePool": "Доступный пул",
-            "balance": "Баланс",
+            "balance": {
+                "title": "Surf stakes in {0}",
+                "note": "{0} for all time",
+                "reward": "+{0} % or {1} {2}"
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "reward": "+{0} % или {1} {2}"
```

File: src/ru.json [hunk 36]
```diff
@@ -1195,33 +1195,50 @@
...
         },
         "stakingScreen": {
-            "about": "Информация",
-            "advancedFeatures": "Расширенные функции",
-            "age": "Возраст",
-            "assurance": "Гарантия",
-            "availablePool": "Доступный пул",
-            "balance": "Баланс",
+            "balance": {
+                "title": "Surf stakes in {0}",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "title": "Хранится в {0}",
```

File: src/ru.json [hunk 37]
```diff
@@ -1195,33 +1195,50 @@
...
+                "note": "{0} for all time",
```
Comment (AnnStepanova) [inline src/ru.json]:
```suggestion
                "note": "{0} за все время",
```

**Review by sertony — APPROVED** [review]

**Pull request closed without merging**